components/grid/selection/cells.md

```diff
@@ -0,0 +1,214 @@
+---
+title: Cell Selection
+page_title: Grid - Cells Selection
+description: Learn how to select cell  in Blazor Grid component. Explore the selected cells. Discover cell selection bevahior when combined with other Grid features. Try the practical sample code for cell selection.
+slug: components/grid/selection/cells
```

Comment:
Use the new slug format, in this case, it is grid-selection-cell (component-feature-subfeature).